low-level/files/opt/seagate/sspl/setup/generate_resource_health_view/resource_health_view

```diff
@@ -1302,16 +1302,23 @@ if __name__ == "__main__":
         print('Missing arguments')
         parser.print_help()
         sys.exit(1)
+    storage_type = Conf.get(GLOBAL_CONF, f"{STORAGE}>{ENCLOSURE}>{TYPE}",'virtual').lower()
+    server_type = Conf.get(GLOBAL_CONF, f"{CLUSTER}>{SRVNODE}>{NODE_TYPE}",'virtual').lower()
     print(f"Storage Type : '{storage_type}'")
     print(f"Server Type '{server_type}'")
 
+    if storage_type in ['5u84', 'rbod', 'pods']:
+        storage_type = 'rbod'
+
     if storage_type == 'virtual' and server_type == 'virtual':
         print("Resource health view is not supported in virtual server and virtual storage deployment")
         print("Exiting ...")
         sys.exit(os.EX_USAGE)
 
-    sys_site_id, sys_rack_id, sys_node_id, sys_cluster_id, sys_data_path = \
-        SSPLHealthView._update_required_data_to_consul(args)
+    if SSPL_STORE_TYPE == "consul":
+        sys_site_id, sys_rack_id, sys_node_id, sys_cluster_id, sys_data_path = \
+            SSPLHealthView._update_required_data_to_consul(args)
```

Comment:
if store type is not consul, then where are we updating these required data? Ignoring?

Reply:
Anyways resource health view script is taking values from confstore not instead of consul. So no use of this function/code.
Removed unused code from the script now.
Done.

Reply:
This may be required for backward compatible. I'm not sure. Keeping that unused code is just fine. I tried to get clarified on the condition of store_type.

Reply:
This code has logic to just get required data from common config consul and update to the SSPL config for login realstore enclosure with proper credentials. But now we are using confstore to those data.
I don't think so that code will be needed in the future for the resource health view script.
@cdeshmukh @thavanathan needs your input here.

Reply:
Thats fine, backward compatibility is at lower priority. we can do away in independent script like these, as its easier to handle if at all needed

Reply:
Done